### DISCUSSION TITLE
Transfer Issues Between Organizations

### DISCUSSION THREAD

**Discussion by michael-hawker** [community/community#5766] [Projects and Issues]
I work across a few different organizations where it'd be nice to be able to transfer issues between them.

For instance our doc repo is in one org and our code repository is in another. We also have partner teams we work with that are in different orgs. We sometimes discover that an issue may be external to our product and it'd be nice to transfer directly to our partner to maintain the history as one issue with the context (and vice-versa).

It'd be great if someone has write access to both repositories that an issue could be transferred between them still.

**Comment by haarchri** [comment]
We have the same between crossplane org and crossplane-contrib org

**Comment by porcelli** [comment]
As part of an organization (ie. IBM) that makes several contributions to several different repositories from multiple organizations, I'd like to have more portability between issues created in one Organization to be transferred to a different Organization.

Practical example:

I work for IBM, and my group uses [GitHub Projects](https://github.com/orgs/IBM/projects/21) to track our plans and the progress of our open-source contributions to the [KIE group](https://github.com/kiegroup). We're using an [IBM repository](https://github.com/IBM/kie-roadmap) to track work scoped only to IBM interest.
However, after some discussions, we see that some issues initially scoped only to IBM's interest have some traction with the broader KIE group open-source community.
In such cases, we'd like to transfer an issue initially under the IBM repository to the [KIE group repository](https://github.com/kiegroup/kie-issues) instead of having to copy-n-paste and close.

**Comment by reinecke** [comment]
We also have use cases for this. Our [main repository ](https://github.com/AcademySoftwareFoundation/OpenTimelineIO) is hosted in one organization, but we're currently spinning off components from the main repo to several repositories in [another organization](https://github.com/OpenTimelineIO).

It would be really helpful to be able to gracefully migrate issues related to those components to their new repos.

**Comment by diogomart** [comment]
We'd also benefit from transferring issues across organizations.

**Comment by rhornig** [comment]
We also have use-cases where users report issues in wrong repository (different parts of our software ecosystem are handled by different organisations)

**Comment by fgnievinski** [comment]
As a workaround, [Kamino](https://github.com/gatewayapps/kamino) can be used to clone an issue to another repository, even across different organizations; then the original issue can be closed, including a link to the cloned issue.

**Comment by erikbethke** [comment]
I created this script to manage this for our organization - hopefully it is helpful you:

[IMP](https://github.com/MillionOnMars/issueMigratorPro)

**Comment by thientruongats** [comment]
- Create an temporary repo in the source org.
- Transfer the issues from the source repo to the temporary repo.
- Transfer the temporary repo to the destination org.
- Transfer the issues from the temporary repo to the destination repo.
- Delete the temporary repo.

**Reply by headius** [reply]
Slimy but satisfying.

**Comment by Neustradamus** [comment]
Dear @github @community team,

This ticket is really important.

Thanks in advance.

It is linked to:
- https://github.com/orgs/community/discussions/9695

**Comment by headius** [comment]
Also need this to work on issues and PRs in private organization but transfer to public organization when ready for consumption.
